TestObjectiveDropbox/IntegrationTests/TestClasses.m

```diff
@@ -2233,7 +2258,11 @@ @implementation TestFormat
 + (void)abort:(DBRequestError *)error routeError:(id)routeError {
     [self printErrors:error routeError:routeError];
     MyLog(@"Terminating....\n");
-    exit(0);
+    NSException* myException = [NSException
```

Comment:
I originally made this change so that I can try catch failures in tests but because these are async I wasn't able to. I'm keeping it as exception because the error message is better